logo/LICENSE

```diff
@@ -0,0 +1 @@
+# The kind logo files are licensed under a choice of either Apache-2.0 or CC-BY-4.0 (Creative Commons Attribution 4.0 International).
```

Comment:
Not really sure on conventions around logo licensing... does it need a separate license?

Reply:
i just copied the license from k/k and `s/Kubernetes/kind/`

this was the discussion around this file in k/k:
https://github.com/kubernetes/kubernetes/pull/67010

Reply:
Thanks, I totally missed this previously